pkg/cmd/kafka/topic/produce/produce.go

```diff
@@ -0,0 +1,133 @@
+package produce
+
+import (
+	"context"
+	"os"
+
+	kafkaflagutil "github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/flagutil"
+	kafkainstanceclient "github.com/redhat-developer/app-services-sdk-go/kafkainstance/apiv1internal/client"
+
+	"bufio"
+	"io/ioutil"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/kafkacmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/core/logging"
+	"github.com/redhat-developer/app-services-cli/pkg/core/servicecontext"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/contextutil"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	topicName string
+	kafkaID   string
+	key       string
+	partition int32
+
+	IO             *iostreams.IOStreams
```

Comment:
FYI - we are trying to avoid such verbose format/boilerplate in new commands and using factory 
as input:

https://github.com/redhat-developer/app-services-cli/blob/main/pkg/cmd/connector/namespace/list/list.go#L32